flink-core/src/main/java/org/apache/flink/util/StringValueUtils.java

```diff
@@ -114,15 +114,17 @@ public boolean next(StringValue target) {
 			int pos = this.pos;
 			
 			// skip the delimiter
-			for (; pos < limit && Character.isWhitespace(data[pos]); pos++);
+			for (; pos < limit && Character.isWhitespace(data[pos]); pos++) {
```

Comment:
The `NeedBraces` module was extended with the strict-checkstyle to also apply to `do`, `while` and `for`.

Since i can't selectively disable parts of the module I had to make the changes instead.